packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/tests/index.test.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import ListView from '../index';
+
+const history = createMemoryHistory();
+
+const App = (
+  <ThemeProvider theme={lightTheme}>
+    <IntlProvider locale="en" messages={{}} defaultLocale="en" textComponent="span">
+      <Router history={history}>
+        <ListView />
+      </Router>
+    </IntlProvider>
+  </ThemeProvider>
+);
+
+describe('ADMIN | Pages | AUDIT LOGS | ListView', () => {
+  it('renders and matches the snapshot', () => {
+    const {
+      container: { firstChild },
+    } = render(App);
+
+    expect(firstChild).toMatchInlineSnapshot(`
```

Comment:
What do you think about removing the snapshot test and replace it with more granular tests (e.g. check if the title is properly rendered)? In the end this page will become big (markup-wise) and I would find it more useful to have smaller tests ...

Reply:
sure, yes I also think so, that's a better way of testing and snapshots should be for very specific formats. But I just tried to follow other pages.